### ISSUE TITLE
Players keep disconnecting

### ISSUE THREAD

**Issue by YQCNM** [GeyserMC/Geyser#4620]
### Describe the bug

After entering the server, the player will be disconnected from the server within 10-30 seconds, and the server's background terminal will display “Keystone Edition Client Connection Timeout”.

### To Reproduce

1. Join the server
2. Wait a while
3. Disconnected

### Expected behaviour

Players play normally

### Server Version and Plugins

Checking version, please wait...
[15:13:05 INFO]: This server is running Paper version git-Paper-318 (MC: 1.20.2) (Implementing API version 1.20.2-R0.1-SNAPSHOT) (Git: 9271ee7)
You are running the latest version
Previous version: git-Paper-241 (MC: 1.20.2)

Plugins:
Geyser
Floodgate
ViaVersion 4.10

### Geyser Dump

https://dump.geysermc.org/aHLc4px8cJVAEhRoEyvcJkYuMNtEN6E0

### Geyser Version

2.2.3-SNAPSHOT (git-master-9cb9a14)

### Additional Context

I'm using DeepL for the translation, so please forgive me if anything is unclear.

**Comment by YQCNM** [comment]
*Bedrock Edition Client Connection Timeout

**Comment by duronald** [comment]
I’m also having this issue. Tried fiddling with the forward ping setting in the config since that seemed to be an issue in previous builds but no dice.

**Comment by YQCNM** [comment]
I figured it out, it seems to be a VM issue. It was originally running on an Ubuntu VM, but after running it outside of the VM, it was fine.

**Comment by duronald** [comment]
Oh wow, I’ve been running it inside a VM too. Are you running it bare metal now? I’m not willing to run it bare metal due to security concerns.

**Comment by YQCNM** [comment]
Yes, because the server is played with a few of my friends and is not made public, so I am not worried about security (for now), Anyway, I haven't had a problem since I didn't run it on a virtual machine

**Comment by duronald** [comment]
Got it. Will try tomorrow in some different configurations and see if the issue persists.

**Comment by duronald** [comment]
@YQCNM What specific hypervisor and networking configuration are you using? I was using VMWare with their provided bridging configuration before. Going to try QEMU/KVM and Docker.

**Comment by duronald** [comment]
I resolved the issue moving it into a QEMU KVM instance away from VMWare. Not sure why this is happening since Geyser / Floodgate should be networking driver agnostic but the issue can be closed.
